src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -139,6 +142,22 @@ public function iSpecifyFilterDateToAs(string $dateTime): void
         $this->client->addFilter('checkoutCompletedAt[before]', $dateTime);
     }
 
+    /**
+     * @When I resend the order confirmation email
+     */
+    public function iResendTheOrderConfirmationEmail(): void
+    {
+        /** @var RequestInterface $request */
+        $request = $this->requestFactory->custom(
+            \sprintf('%s/admin/orders/resend-order-confirmation-email', $this->apiUrlPrefix),
+            HttpRequest::METHOD_POST,
+        );
+
+        $request->setContent(['orderToken' => $this->sharedStorage->get('order')->getTokenValue()]);
+
+        $this->client->executeCustomRequest($request);
```

Comment:
You should be able to use here `$this->client->customItemAction(...)` and not have to add an API prefix to the context